source/main.cpp

```diff
@@ -214,7 +214,11 @@ int main(int argc, char *argv[])
 		if(!isFullscreen)
 			SDL_GetWindowSize(window, &windowWidth, &windowHeight);
 		
-		
+		// menuPanels is used for the panels related to pilot creation, preferences,
+		// game loading and game saving.
+		// gamePanels is used for the main-panel (flying your spaceship), the panet
+		// dialog and all other game-content related dialogs are placed on top of them.
+		// If there are menuPanels, then they are shown before gamePanels.
```

Comment:
before -> in front of?

Reply:
Yes, it looks to the user as if the menuPanels are on top of the gamePanels. But the game internally switches between 2 stacks of panels. So technically they are not in front of the other panels. I added some text where I indicated that menuPanels take priority over gamePanels if both stacks exist.